src/Engine/GUI/CodeEditor.cpp

```
@@ -897,6 +1313,397 @@ void CodeEditor::Render()
+	if (mShowFindReplaceWindow) {
+		ImVec2 cursorPos = ImGui::GetCursorPos();
+		ImVec2 parentPos = ImGui::GetWindowPos();
+		ImVec2 parentSize = ImGui::GetWindowSize();
+
+		float padding = 20.0f;
+
+		static const std::string currentDocumentComboStr = "Current document";
+		static const std::string selectionComboStr = "Selection";
+		static int selectedSearchRange = 0;
+
+		const char *searchRanges[] = { currentDocumentComboStr.c_str(), selectionComboStr.c_str() };
```

Clang (14.0.6)

[warning] do not declare C-style arrays, use std::array<> instead

(modernize-avoid-c-arrays)